appengine/cloudsql_postgresql/README.md

```diff
@@ -1,15 +1,17 @@
-# Cloud SQL for MySQL Node.js sample on App Engine flexible environment
+# Cloud SQL for PostgresSQL Node.js sample on App Engine flexible environment
```

Comment:
Nit: IIRC, it's `PostgreSQL` (extra S).

Reply:
Me and my fat fingers :( It's been fixed